.travis.yml

```diff
@@ -21,6 +21,7 @@ before_install:
   - if [[ "$BOARD" =~ "arduino:samd:" ]]; then
       arduino --install-boards arduino:samd;
       arduino --install-library WiFi101;
```

Comment:
An interesting thing is the first library install always failed with message "selected libraries not found", no idea why this issue happen on travis